elements/bulbs-clickventure/clickventure.js

```diff
@@ -215,4 +220,10 @@ export default class Clickventure {
       this.showNewNode(nodeId, transition);
     }
   }
+
+  adRefresh () {
+    if (this.nodeClickCount % 5 === 0) {
+      this.adsManager.reloadAds();
```

Comment:
Let's explicit reload just the sidebar slot for now: 

```
let sideAd = $('.dfp-slot-sidebar-primary');

// should always be there but just in case...
if (sideAd.length > 0) { 
  this.adsManager.unloadAds(sideAd);
  this.adsManager.loadAds(sideAd);
}
```